binarytree/__init__.py

```diff
@@ -365,6 +365,60 @@ def get_parent_node(root, node):
     return None
 
 
+def merge_trees(node1, node2):
```

Comment:
For now only the `merge_trees` function is used. But in my opinion, if we consider the tree merge as a property of `Node class` it will look nice. (eg. We can just type `node3 = node1 + node2` thus `node3 = merge_trees(node1, node2)` will be excuted in the backend by using just an `__add__` magic function of python.)